src/mcts/node.h

```diff
@@ -80,11 +80,11 @@ class Node {
       return q_;
     }
   }
-  // Returns U / (Puct * N[parent])
+  // Returns U / N[parent]
```

Comment:
```
U = cpuct * p * sqrt(N[parent]) / N.
```
This function returns `p / N`,
or
```
U / (puct * sqrt(N[parent]))
```

But it seems that indeed there was a mistake (missing sqrt).

Reply:
What's the correct comment?

Reply:
```cpp
// Returns p / N = U / (cpuct * sqrt(N[parent]))
```
probably.

Reply:
Ah, gotcha. That's exactly the purpose the comment should serve :)

Reply:
Shouldn't it be called `ReducedU` or something like that? Otherwise people new to the code won't necessarily grasp that a priori, and then writing code similar to

```oss << "(U: " << std::setw(6) << std::setprecision(5)
        << node->GetU() * kCpuct *
               std::sqrt(std::max(node->GetParent()->GetChildrenVisits(), 1u))```

will be extremely prone to bugs.

Reply:
It should indeed be called `GetUPartial()`.
Feel free to change it in this PR, or I can do that later.